<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCM</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="inputGroup">
        <property name="statusTip">
         <string>输入的音频的参数，如果为纯ＰＣＭ码流，则需要自己手动填写</string>
        </property>
        <property name="title">
         <string>input parameter</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QGroupBox" name="srcRateGroup">
           <property name="title">
            <string>SampleRate</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QRadioButton" name="srcRate22050">
              <property name="text">
               <string>22050</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="srcRate44100">
              <property name="text">
               <string>44100</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="srcRate48000">
              <property name="text">
               <string>48000</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="srcFormatGroup">
           <property name="title">
            <string>format</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QRadioButton" name="srcUint8">
              <property name="text">
               <string>unsigned 8 bits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="srcInt16">
              <property name="text">
               <string>signed 16 bits</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="srcInt32">
              <property name="text">
               <string>signed 32 bits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="srcFloat">
              <property name="text">
               <string>float</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="srcDouble">
              <property name="text">
               <string>double</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="srcChannelsGroup">
           <property name="title">
            <string>channels</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QRadioButton" name="srcChannels1">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="srcChannels2">
              <property name="text">
               <string>2</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="outputGroup">
        <property name="statusTip">
         <string>如果不需要重新采样，则选择一样的参数</string>
        </property>
        <property name="title">
         <string>output parameter</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QGroupBox" name="dstRateGroup">
           <property name="title">
            <string>SampleRate</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QRadioButton" name="dstRate22050">
              <property name="text">
               <string>22050</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="dstRate44100">
              <property name="text">
               <string>44100</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="dstRate48000">
              <property name="text">
               <string>48000</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="dstFormatGroup">
           <property name="title">
            <string>format</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QRadioButton" name="dstUint8">
              <property name="text">
               <string>unsigned 8 bits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="dstInt16">
              <property name="text">
               <string>signed 16 bits</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="dstInt32">
              <property name="text">
               <string>signed 32 bits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="dstFloat">
              <property name="text">
               <string>float</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="dstDouble">
              <property name="text">
               <string>double</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="dstChannelsGroup">
           <property name="title">
            <string>channels</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QRadioButton" name="dstChannels1">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="dstChannels2">
              <property name="text">
               <string>2</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextEdit" name="debugText">
          <property name="statusTip">
           <string>信息提示窗口</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="statusTip">
         <string>选择文件的封装格式</string>
        </property>
        <property name="title">
         <string>Output File Type</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QRadioButton" name="pcmTypeButton">
           <property name="text">
            <string>PCM</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="wavTypeButton">
           <property name="text">
            <string>WAV</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="mp3TypeButotn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>MP3</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>File Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="srcPathLineEdit">
        <property name="statusTip">
         <string>选择要操作的音频文件</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pathSelectButton">
        <property name="text">
         <string>select file</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="statusTip">
         <string>播放源文件</string>
        </property>
        <property name="text">
         <string>play</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="testButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="statusTip">
         <string>测试用，试听转换后的音频，暂时保留该功能</string>
        </property>
        <property name="text">
         <string>test</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
